tests/unit/test_awsrequest.py

```diff
@@ -72,6 +72,31 @@ def seek(self, offset, whence):
         raise ValueError("Underlying stream does not support seeking.")
 
 
+class Seekable(object):
+    """This class represents a bare-bones,seekable file-like object
+
+    Note it does not include some of the other attributes of other
+    file-like objects such as StringIO's getvalue() and file object's fileno
+    property. If the file-like object does not have either of these attributes
+    requests will not calculate the content length even though it is still
+    possible to calculate it.
+    """
+    def __init__(self, stream):
+        self._stream = stream
+
+    def __iter__(self):
+        yield self._stream.__iter__()
```

Comment:
Yeah that's right I need to return the iterator. Was not paying too closely to it. I only need this to make the test object look like an iterable.